armi/reactor/grids/grid.py

```diff
@@ -158,6 +158,10 @@ def __setstate__(self, state: Dict):
     def isAxialOnly(self) -> bool:
         """Indicate to parts of ARMI if this Grid handles only axial cells."""
 
+    @property
+    def cornersUp(self) -> bool:
+        return False
+
```

Comment:
`CartesianGrid.cornersUp` doesn't make sense to me. This seems like putting hex specific logic onto the base class just to unify the API.